database/src/migrations/release.0.19/biohub.sql

```diff
@@ -1368,88 +1365,6 @@ COMMENT ON COLUMN proprietor_type.revision_count IS 'Revision count used for con
 COMMENT ON TABLE proprietor_type IS 'Identifies the available reasons that subject data can be proprietary.'
 ;
 
--- 
--- TABLE: security 
---
-
-CREATE TABLE security(
```

Comment:
Just checking in case: is this security stuff supposed to be removed? Or is this a weird merge? I know Roland is currently working on more security related changes, so might be good to sync up in case anything in this PR clashes with anything in his local work.  I've been away so if you are already in sync then ignore me!

Reply:
Roland requested we pull the security stuff as the implementation may still change significantly.

Reply:
Yeah, this is in line with what we discussed.